fixture-monkey-jakarta-validation/src/main/java/com/navercorp/fixturemonkey/jakarta/validation/generator/JakartaValidationNullInjectGenerator.java

```diff
@@ -73,8 +75,45 @@ public double generate(ObjectPropertyGeneratorContext context) {
 			}
 		}
 
-		if (context.isContainer() && annotations.contains(NotEmpty.class)) {
-			return 0.0d;
+		if (context.isContainer()) {
+			if (annotations.contains(NotEmpty.class)) {
+				return 0.0d;
+			}
+
+			Size sizeAnnotation = context.getProperty().getAnnotation(Size.class).orElse(null);
+			if (sizeAnnotation != null && sizeAnnotation.min() > 0) {
+				return 0.0d;
+			}
+		}
+
+		if (context.getOwnerProperty() != null) {
+			Property elementProperty = context.getOwnerElementProperty();
+			if (elementProperty != null) {
+				Set<Class<? extends Annotation>> elementAnnotations = elementProperty.getAnnotations().stream()
+					.map(Annotation::annotationType)
+					.collect(toSet());
+
+				if (elementAnnotations.contains(NotNull.class)) {
+					return 0.0d;
+				}
+
+				if (Types.getActualType(elementProperty.getType()) == String.class) {
+					if (elementAnnotations.contains(NotBlank.class) || elementAnnotations.contains(NotEmpty.class)) {
+						return 0.0d;
+					}
+				}
+
+				if (context.getOwnerProperty().isContainer()) {
+					if (elementAnnotations.contains(NotEmpty.class)) {
+						return 0.0d;
+					}
+
+					Size elementSizeAnnotation = elementProperty.getAnnotation(Size.class).orElse(null);
+					if (elementSizeAnnotation != null && elementSizeAnnotation.min() > 0) {
+						return 0.0d;
+					}
```

Comment:
The following comment suggests that it could be null.
As far as I know, the `@Size` annotation is not related to nullability.

![image](https://github.com/user-attachments/assets/3556fe03-c638-49ea-9d4d-682820396494)